tasks/register/behatRegister.js

```diff
@@ -0,0 +1,54 @@
+module.exports = function (grunt) {
+
+    // Point to local Behat default Profile
+    grunt.registerTask('behatLocal', 'Execute Behat for BDD testing', function () {
```

Comment:
Shouldn't this configuration be as separate targets in the behat.js file /tasks/config?